tests/resources/queue-event-fail.json

```diff
@@ -199,6 +199,14 @@
+    },
+    {
+      "messageId": "g48c54a0-7027-4e37-b7e8-c8d231511c680",
+      "receiptHandle": "AQEBJcBvTRZ1W2LSaUJ0g0ELXlqA8WCL4zJxO63wu0YOVhx44xxxPhsnc+/Q9+1vOPYO+3HupEjXzGRSvfPY5rEEJkgCJe4/RQ+q2kU5LsmJEr1qE/CTdIYe5X/75XeMQ523KKpdNsD9tRhyvEpPpSu50byGbz7J0JyR6lu1E6Q4YuB4QNm+ev1obPMLdEt8RhgvIi/NfEfQf0L1r3TPi3wLho1R61PllPm27He8/1CjCnMyWBzgX+DCjJ7vyRXObMZ/MbhMBKbYpeTcejsKpYX//PPr1yvldp1YPC0wPKp+iqmWxoDDeHXbo8xYRFXDA8rnY5RfkwxxffH7o534vYn8FCZEtqybQuo7pumu6Ah9PsC05tP38syU71ltasljGIA35BgCdSO+9r5rTaBnbO9++Q==",
+      "body": "{\"testerStaffId\":\"1\",\"testStartTimestamp\":\"2024-01-26T14:50:44.279Z\",\"odometerReadingUnits\":\"kilometres\",\"testEndTimestamp\":\"2024-01-26T15:02:10.761Z\",\"testStatus\":\"submitted\",\"make\":null,\"model\":null,\"bodyType\":{\"code\":null,\"description\":null},\"testTypes\":{\"testNumber\":\"W01A00310\",\"createdAt\":\"2024-01-26T14:50:44.279Z\",\"additionalCommentsForAbandon\":null,\"additionalNotesRecorded\":null,\"certificateNumber\":\"4353\",\"customDefects\":[],\"requiredStandards\":[{\"sectionNumber\":\"6a\",\"sectionDescription\":\"Lighting\",\"rsNumber\":2,\"requiredStandard\":\"An obligatory (or optional) lamp or reflector;  incorrect number fitted\",\"refCalculation\":\"6.2a\",\"additionalInfo\":true,\"inspectionTypes\":[\"normal\",\"basic\"],\"prs\":false,\"additionalNotes\":\"The bulbs were slightly worn\"},{\"sectionNumber\":\"01\",\"sectionDescription\":\"Noise\",\"rsNumber\":1,\"requiredStandard\":\"The exhaust must be securely mounted\",\"refCalculation\":\"1.1\",\"additionalInfo\":true,\"inspectionTypes\":[\"normal\",\"basic\"],\"prs\":false,\"additionalNotes\":\"The exhaust was held on with blue tac\"}],\"defects\":[],\"emissionStandard\":null,\"fuelType\":null,\"modificationTypeUsed\":null,\"modType\":null,\"name\":\"Specialist test\",\"particulateTrapFitted\":null,\"particulateTrapSerialNumber\":null,\"prohibitionIssued\":false,\"reasonForAbandoning\":null,\"secondaryCertificateNumber\":null,\"smokeTestKLimitApplied\":null,\"testExpiryDate\":null,\"testResult\":\"fail\",\"testTypeEndTimestamp\":\"2024-03-04T10:19:00.907Z\",\"testTypeId\":\"125\",\"testTypeName\":\"Basic IVA inspection\",\"testTypeStartTimestamp\":\"2024-03-04T10:18:44.191Z\"},\"vehicleClass\":{\"code\":\"v\",\"description\":\"heavy goods vehicle\"},\"testResultId\":\"1\",\"vehicleSize\":\"small\",\"vin\":\"P0123010956789\",\"testStationName\":\"Abshire-Kub\",\"vehicleId\":\"ZX345CV\",\"countryOfRegistration\":\"gb\",\"vehicleType\":\"hgv\",\"preparerId\":\"AK4434\",\"preparerName\":\"Durrell Vehicles Limited\",\"odometerReading\":12312,\"vehicleConfiguration\":\"rigid\",\"testStationType\":\"gvts\",\"testerName\":\"CVS Dev1\",\"vrm\":\"ZX345CV\",\"testStationPNumber\":\"09-4129632\",\"numberOfSeats\":50,\"testerEmailAddress\":\"cvs.dev1@dvsagov.onmicrosoft.com\",\"euVehicleCategory\":\"l1e-a\",\"order\":{\"current\":2,\"total\":2}}",      "md5OfBody": "9586727cbc9f3312542387099b60982c",
+      "eventSource": "aws:sqs",
```

Comment:
Is the below supposed to be in there?

`"md5OfBody": "9586727cbc9f3312542387099b60982c",`